ACKategories-iOSTests/FlowCoordinator/Dummies.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  Dummies.swift
+//  ACKategories-iOSTests
+//
+//  Created by Lukáš Hromadník on 23.11.2020.
+//
+
+import UIKit
+@testable import ACKategories
+
+extension Base.FlowCoordinatorNoDeepLink {
+    func push(_ vc: UIViewController) {
+        navigationController?.pushViewController(vc, animated: false)
+    }
+    
+    func present(_ vc: UIViewController) {
+        rootViewController.present(vc, animated: false)
+    }
+}
+
+class NavigationFC: Base.FlowCoordinatorNoDeepLink {
+    override func start(in window: UIWindow) {
+        let rootVC = UIViewController()
+        let navVC = UINavigationController(rootViewController: rootVC)
+        rootViewController = rootVC
+        navigationController = navVC
+        
+        window.rootViewController = navigationController
+        window.makeKeyAndVisible()
+        _ = navVC.view
+        
+        super.start(in: window)
```

Comment:
shouldn't we call `super` as the first thing in this method? (same applies to overrides below)

Reply:
👍